src/reaper_osara.cpp

```diff
@@ -4210,6 +4210,22 @@ void cmdSelectFromCursorToStartOfProject(Command* cmd) {
 	outputMessage(translate("selected to start of project"));
 }
 
+void cmdSetPhaseNormalAllTracks(Command* cmd) {
+	int count = CountTracks(nullptr);
+	if (count == 0) {
+		outputMessage(translate("no tracks"));
+		return;
+	}
+	int phaseVal = 0;
```

Comment:
1. This will work, but B_PHASE is actually a bool, not an int, so we should use a bool.
2. As it happens, OSARA already defines a bool called bFalse for this purpose, so we can just use that.

```suggestion
```